w1r3/java/src/main/java/W1R3.java

```diff
@@ -194,11 +170,10 @@ private void worker(
       Transport[] transports,
       Uploader[] uploaders,
       byte[] randomData,
-      int seed,
+      Random random,
```

Comment:
Seems unrelated, but meh.

Reply:
With not having multiple workers, this worker can now share the base SecureRandom from the caller of this method.